bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -390,25 +394,10 @@ class VideoProvider extends Component {
     } catch (error) {
       this.logger('error', 'Video provider failed to fetch ice servers, using default');
     } finally {
-      const videoConstraints = {
-        width: {
-          min: 320,
-          max: 640,
-        },
-        height: {
-          min: 180,
-          max: 480,
-        },
-      };
-
-      if (!navigator.userAgent.match(/Version\/[\d\.]+.*Safari/)) {
-        videoConstraints.frameRate = { min: 5, ideal: 10 };
```

Comment:
I do not recall why we have this chunk but should we keep it?

Reply:
I removed the FPS constraints because the new target bitrate option already takes care of it, so this isn't needed anymore.

Reply:
And the chunk was there because Safari don't support `ideal` constraints and adapter wouldn't take care of it properly.